tests/strings/jsfmt.spec.js

```diff
@@ -1,2 +1,15 @@
 run_spec(__dirname);
 run_spec(__dirname, { trailingComma: "all" });
+
+/**
+ * TS currently broken
+ *
+ * - strings.js
+ *
+ * DIFF:
+ *    -  '\\uD801\\uDC28',
+ *    +
+ *    +  "\\uD801\\uDC28",
```

Comment:
Not sure if this is it but there is a workaround in the flow parser about this. You should try to compare it against babylon and see if you match.